src/Statistics.jl

```diff
@@ -75,15 +75,15 @@ between two iterates is small. For more stopping criteria check the
 
 The algorithm is further described in [[Afsari et al, 2013]](#Afsari2013).
```

Comment:
How does this render in REPL? In `Documenter` this refers to the literature hopefully available at the end of the statistics page (otherwise the link is not valid) but in REPL?. For this situation I like a (maybe shortened a little bit) reference in a block quote (`>`s)

Reply:
It's very unattractive in REPL. I can switch it back. The block quotes look nice.